.github/workflows/e2e-android.yml

```diff
@@ -101,6 +102,10 @@ jobs:
       - name: Check E2E wallet balance
         run: NODE_OPTIONS='--unhandled-rejections=strict' yarn ts-node ./e2e/scripts/check-e2e-wallet-balance.ts
         working-directory: apps/example
+      - name: Override ANDROID_SDK_ROOT
```

Comment:
nit: could we put this in the existing “set Env” step?

Reply:
Changed in 8c8606a591dcec9efce8e2f82f6b17d913651a00!